docs/dapp/sapphire/security.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Security recommendations for Sapphire
+---
+
+# Security
+
+Work in Progress
```

Comment:
I would mention "storage I/O pattern" and "gas usage pattern" here. Maybe some other pattern?